spec/ordinary-and-exotic-object-behaviours.html

```diff
@@ -251,8 +251,7 @@ <h1>[[DefineOwnProperty]] ( _P_, _Desc_ )</h1>
           1. If Type(_P_) is a Symbol, return *false*.
           1. If _P_ is *"length"*, then
             1. Let _lengthDesc_ be ! _T_.[[GetOwnProperty]](_P_).
-            1. Let _extensible_ be _T_.[[Extensible]].
-            1. Return IsCompatiblePropertyDescriptor(_extensible_, _Desc_, _lengthDesc_).
+            1. Return IsCompatiblePropertyDescriptor(*false*, _Desc_, _lengthDesc_).
```

Comment:
unrelated. Tidy up while here, `extensible` is always false, looking it up implies it's dynamic.